README.md

```diff
@@ -154,16 +153,18 @@ $ pip install -r requirements.txt
 
 Since the periodic tasks would need to store the fetched quotes, we need to
 connect to QuestDB. Therefore, we create a new file in the `app` package, called
-`db.py`. This file contains the `SQLAlchemy` engine that will serve as the base
+`db.py`. This file contains the Postgres Pool that will serve as the base
```

Comment:
Actually, it is inconsistently used everywhere (in almost every article I can read), so based on your suggestion, I updated it to be "PostgreSQL connection pool", since we define the pool itself, not the pooling (as the connection reuse mechanism).